get_era5.py

```diff
@@ -0,0 +1,157 @@
+import cdsapi
```

Comment:
Shall we place `get_era5.py` in `src/wam2layers/download/era5.py` or `src/wam2layers/utils/download_era5.py`?

Reply:
I have moved it to src/wam2layers/utils/download_era5.py